main/settings.py

```diff
@@ -9,7 +9,7 @@
 
 DEBUG = "DEVELOPMENT" in os.environ
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['127.0.0.1']
```

Comment:
Please remove this for the PR. You can add the host (or just an asterisk) to the `SITE_NAME` variable in the `.env` file to do this.